exercises/saddle-points/canonical-data.json

```diff
@@ -117,6 +117,68 @@
           "column": 2
         }
       ]
+    },
+    {
+        "description": "Can identify saddle points in a non square matrix",
+        "property": "saddlePoints",
+        "input": {
+            "matrix": [
+                [3, 1, 3],
+                [3, 2, 4]
+            ]
+        },
+        "expected": [
+            {
+                "row": 0,
+                "column": 2
+            },
+            {
+                "row": 0,
+                "column": 0
+            }
+        ]
+    },
+    {
+        "description": "Can identify that saddle points in a single column matrix are those with the minimum value",
+        "property": "saddlePoints",
+        "input": {
+            "matrix": [
+                [2],
+                [1],
+                [4],
+                [1]
+            ]
+        },
+        "expected": [
+            {
+                "row": 1,
+                "column": 0
+            },
+            {
+                "row": 3,
+                "column": 0
+            }
+        ]
+    },
+    {
+        "description": "Can identify that saddle points in a single row matrix are those with the maximum value",
+        "property": "saddlePoints",
+        "input": {
+            "matrix": [
+                [2, 5, 3, 5]
+            ]
+        },
+        "expected": [
+            {
+                "row": 0,
+                "column": 1
+            },
+            {
+                "row": 0,
+                "column": 3
+            }
+        ]
```

Comment:
A cosmetic request.  Please adjust your indentation spacing to conform to the other test cases.